lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py

```diff
@@ -101,12 +101,8 @@ def test_3485_timer_for_dedicated_bearer_with_mme_restart(self):
             )
 
             print('***** Restarting MME service on gateway')
-            self._s1ap_wrapper.magmad_util.restart_services(['mme'])
-
             wait_for_restart = 20
-            for j in range(wait_for_restart):
-                print('Waiting for', j, 'seconds')
-                time.sleep(1)
+            self._s1ap_wrapper.magmad_util.restart_services(['mme'], wait_for_restart)
```

Comment:
I think. the similar changes are required in other restart test cases like, https://github.com/magma/magma/blob/master/lte/gateway/python/integ_tests/s1aptests/test_no_auth_response_with_mme_restart.py#L81

Reply:
Not necessarily. The function `restart_services` was rewritten in a way that the old usage still works the same way (the `wait_for_restart` argument is not required. If it is not given, it will be set to 0). It might be nice to eventually change it everywhere, for the sake of consistency.

Reply:
Okay got it. Thanks for explaining

Reply:
I adapted all functions to the changed syntax.

Reply:
Thank you

Reply:
@rsarwad, feel free to approve the PR, if you are satisfied with the changes :)